guide/daily-spending-wallet/receiving.md

```diff
@@ -0,0 +1,194 @@
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+{% include picture.html
+   image = "/assets/images/guide/daily-spending-wallet/receiving/CloseWalletNotification.png"
+   retina = "/assets/images/guide/daily-spending-wallet/receiving/CloseWalletNotification@2x.png"
+   modalImage = "/assets/images/guide/daily-spending-wallet/receiving/CloseWalletNotification@2x.png"
+   layout = "shadow"
+   caption = "Notify the user that closing the app could mean that payments can not be received."
+   alt-text = "Screen showing a notification that is asking the user if they want to close their app as it could mean that payments can not be received."
+   width = 250
+   height = 541
+   modalWidth = 250
+   modalHeight = 541
+%}
+
+</div>
+
+## Receiving lightning payments 
+
+Once the user is online, a payment channel with [inbound liquidity]({{ '/guide/how-it-works/liquidity/'| relative_url }}) is also required to receive the payment. If the user already has a payment channel with enough inbound liquidity the payment will be [received]({{ '/guide/daily-spending-wallet/receiving/#received-payment'| relative_url }}).
+
+If the user does not have a channel open, or one with enough inbound liquidity, a new channel needs to be opened. Any additional fees required to open a channel needs to be communicated to users when [creating a payment request]({{ '/guide/daily-spending-wallet/requesting/'| relative_url }}) before they share it.
+
+Channel opens should be done by a lightning service provider (LSP). Channels should be opened [on-demand]({{ '/guide/how-it-works/lightning-services/#on-demand-liquidity'| relative_url }}) and use [zero-confirmation]({{ '/guide/how-it-works/lightning-services/#zero-confirmation'| relative_url }}) so users can receive and spend their bitcoin instantly.
+
+## Receiving on-chain payments
+
+To receive on-chain payments and maintain a single lightning balance a [lightning wallet server (LWS)]({{ '/guide/how-it-works/lightning-services/#what-are-lightning-wallet-servers'| relative_url }}) is required to conduct [submarine swaps](https://blog.muun.com/a-closer-look-at-submarine-swaps-in-the-lightning-network/) on the users behalf. Submarine swaps move bitcoin between on-chain and lightning.
+
+{% include tip/open.html color="blue" icon="info" label="Experimental options" %}
+
+Experimental options such as [Peerswap](https://www.peerswap.dev/) and [Collaborative funding](https://bitcoinops.org/en/topics/dual-funding/) may be more trustless options that could be used to receive on-chain payments whilst maintaining a single lightning balance.
+
+{% include tip/close.html %}
+
+### Refunds
+
+There may be scenarios where a submarine swap is not completed successfully such as the LWS being offline. In these cases the users will need to temporarily have a refunded on-chain balance of which they can claim to an on-chain address they control. 
```

Comment:
```suggestion
There may be scenarios where a submarine swap is not completed successfully, such as when the lightning service provider is offline. In these cases, the users will need to temporarily have a refunded on-chain balance which they can claim to an on-chain address they control. 
```

Reply:
`the users will need to temporarily have a refunded on-chain balance` doesn't read smoothly. What does it mean to temporarily have a balance?

Reply:
Its only there to be claimed, I will re-phrase. The LWS not an LSP does the swaps fyi